### PR TITLE
USHIFT-713: rebase.sh - etcd/

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

File: scripts/auto-rebase/rebase.sh [hunk 1]
```diff
@@ -1029,8 +1039,8 @@ rebase_to() {
         echo "No changes to buildfiles."
     fi
 
-    title "# Removing staging directory"
-    rm -rf "$REPOROOT/_output"
+    # title "# Removing staging directory"
+    # rm -rf "$REPOROOT/_output"
```
Comment (ggiguash) [inline scripts/auto-rebase/rebase.sh:1043]: Should we uncomment these?

File: scripts/auto-rebase/rebase.sh [hunk 2]
```diff
@@ -559,10 +559,17 @@ update_go_mod() {
...
-
-    popd >/dev/null
 }
 
+# Updates go.mod file in dirs defined in GO_MOD_DIRS
+update_go_mods() {
+    for d in "${GO_MOD_DIRS[@]}"; do
+        pushd "${d}" > /dev/null
+        update_go_mod
+        go mod tidy
```
Comment (ggiguash) [inline scripts/auto-rebase/rebase.sh:569]: tidy should be removed from here, or from the function, right?

File: scripts/auto-rebase/rebase.sh [hunk 3]
```diff
@@ -878,6 +885,8 @@ EOF
 
 # Builds a list of the changes for each repository touched in this rebase
 update_changelog() {
+    set -x
```
Comment (ggiguash) [inline scripts/auto-rebase/rebase.sh:888]: Remove this?

File: scripts/auto-rebase/rebase.sh [hunk 4]
```diff
@@ -974,30 +983,31 @@ rebase_to() {
     fi
 
     rebase_branch="rebase-${ver_stream}_amd64-${amd64_date}_arm64-${arm64_date}"
-    git branch -D "${rebase_branch}" || true
-    git checkout -b "${rebase_branch}"
+    # git branch -D "${rebase_branch}" || true
+    # git checkout -b "${rebase_branch}"
```
Comment (ggiguash) [inline scripts/auto-rebase/rebase.sh:987]: Uncomment these?

**Comment by pmtk** [comment]
@ggiguash - addressed comment, please review again

**Pull request merged**